packages/core/src/js/feedback/branding.ts

```diff
@@ -0,0 +1,5 @@
YgediSAiRIWms18f7gTlWEc/TwgKyB/PXPXz5x8h3PskASkcFReyJsiNA5cs2hAdkCZvvzyxGgoEG5d3ZEgKKtVo2v7UcGPLr22IC8/bC8ECzoMwmmvdns8oaEscW73UODtOb4YQE5zOwBoDCBGQWJqkg5ckMCUtz2BIACB+czJfODVwvIwBrxslYepvVGCpA72nu3OOt2LUBSnCS+6xOhQ72wL995OEDQwHiW0rmaNAHJ5R3sTtyMWEXGAwQMishLPAuJV6CiyWLEKjIUIGhAXEuo3kHCQOJpA7y/cyUtb3bqjBwLEEv1uPSPU6AwF+ZHUyru71XB6oQ8tuswgHhUj1fSVgxST0jQKuJZwbBwzR89BCAhcDi/7rCVxCObM3t77Jsf7viOYwDi2Vpd0dgrozIg19rba188ZHNnnB4QJugYF3gFDGWvU6uHtlpe9yBG76w55wckuHrsHeUFCRyoTj9oxbRaZ/+e1qkBgbPxPD88BvyyfGQzlAckTKB67Mt8yu+1L6t39LzTAsIE2ZYNmbnelYSywaHVYvY984X9tIAw1ePyy4DwGjtKPDIrs79HsKItnsdZoysBu/4pAWEC61Yvzay1OcMjcGoFq36f1pMXzwkIejG/05rUhKRWy4Oe2SMZsFk+ct7pAEEdW/pUyQTqs+OMdwP0TF7BilavUi0jA9p77fMBAlYP5JnSAon1blAjWJnzWs/pHeDW9U4FCJppmYzHBM3mJEvwMPsi8N8KJBRMr+plDWyv+acBBIZjmiY2gJhg/X7h+/LHt8/vWeehwcokgEvbmLNaEgGrTdS8cQEx3guYwHl6FuF/tpzZk00C+4BDk8+ZqsgpAKnlQCZgrZCgZ7UA+QIS8G7ntW9UZShd9xyAoM5z+u7SKjILiaUNQT+j8KgizDk99i0N5Khx3QMCZ9S9koZ2x9KCWO8jcLA6nRO9A52h1eofEKJ6vMg2lYJntcFSRdBg9cjmMJjGM0ZVBWTdrgExVY+ASoIGrSXDosFqgXEvVeYZkUCOGts1IKsoXpBYgnVzDhq01gs7Eqwe52PvXF5wRkFwtG73gGyH8wDFI4gYO9gAQoFk97kMIPSMHrrWgOP7XbHW1jH7os67tMLDmUhmt17YkRctj7NteqFn7PXZ9zQVZB/oaGaNgAQJXMuFHU0IHpf1kVqtUwLi0nYZX7fQwLVkdwRGrzZr1RitIpYzxvQb91c9NSDmS7wREjSA2OBF9vEMUqZSs2e8H8oxI04PCNsObHJbHIoGEBu86D6WM12GIQKn9b4Vg8DxqkMAUhMSNIDY4EX2YUG8FUpIi2d92s6GZBhA6NeXp59Lpr+mjmZ3di9kH3aPm4D8+PMn9FcleT0WRAMzHCDM5dKa9ZDsvu7FBA8CiOXV7FZAomf0hjQKlCEBQYPJ4z6CtCEZbRa7x61AZDT1tiECkuYBeRT+zZt3Xn/AchORcagl6yEZlt0HORO7x1EQImfs5cLeNCCvPkswPrteOhf9rMLSaiHBy7ZAGXscAYLuz54zolLcWrNpQG5mJEdQGEiYOwJ692EzPJLFI1qcTD0zQGkWkHtCswF0TVQkqFqvIvd025/fU8P9ush9y6Ln2ICU/iCUQzVhWgO2iiDBw2R45CxRgCA2eDyARILSZAVBsqBXH4vuyQYXUq0y9mAgLAlI5JwtX9jbBKS0emyecqgi6B2BBRPNrkylQoIzChD0nKyeJbBaxjQHCJrJPX/OAHUqG1zRAYycg61SJUGHnLPVKtIUIDAc5KfOXm/5bHAhAcwkAGR99gwlgCB2PK/n1A2U2Fcypi1AKrVWe6FQpzJVBNmDDWDkMYBp40qCax3TQtIrtfXauGYAaUlIpDVoNYCRMzCQI0GH2NJaq9UOIA1Uj83pSIZf5zAZGAkaJoCRhMOsjwCC6tnShb0JQBBnZn2wFB3ASNAwVQpZn7nnIIAwL4TMmVGbSsZXBwSGg8zYJWLsxyCAMM5EArjF9VE9kfNua0dXtpIz9AdI0isH6tDoNotZv/SizgBYElyXY9BkmGXX0VmqAoIKxvb7jDPRtoDJdkiVYgCJXp/RtRTaVqpIX4AkVY/NOQjADCBIlWLWbxEQ5MwtvGjVBQR5uUqGY3UO4kymHUDWZwCJBpypIGhlzu4aLs/UDSBMgLAO3M8rbQmiAWFemloFBEkMtZ98qwECiVShemyQIG0KA3ErADK2WxJQtK4W2/ZzBcgdJaMd2QogTIWyBCFS3bJt6w4Qpn2xOG8/F6l0TBZGAERfshDbs4NQgBREaGn2fF5qnh8KlnUdMi/Lf5Zpele0KNEKIoCsQVxkx37QsnwsmcMkoRXAkrUvxyzz/LFY08p/xq1ai7WKBgPCeCNzDgEIlEkDz8IAAsFtsB2tnIatXk2tCkiWwJ6CHa3FBJkAOfYOo6mnv6sCAvXInqcOWotxpgC54wyiKnu6tyog60HOVEUEiGdoPq1Vs71a968OyJmqiABxBqRy9WgCkMfLOvHr851d4bKcAHGR8WmRBuBoBpCzQCJAnABpBI6mADkDJALEAZCG4GgOkE3eXlsuAcIDsmo3L8vD73/9+oVfxX9m9Uv60ZFWUKBPsv31gVYUIJBcj3/a7jFLNwjGdpKmAcHk1uhsBdivmqx2tlYpbmknQLKjSvt1pYAA6cpdMjZbAQGSrbj260oBAdKVu2RstgICJFtx7deVAgKkK3fJ2GwFBEi24tqvKwUESFfukrHZCgiQbMW1X1cKCJCu3CVjsxUQINmKa7+uFBAgXblLxmYrIECyFdd+XSkgQLpyl4zNVkCAZCuu/bpSQIB05S4Zm62AAMlWXPt1pYAA6cpdMjZbAQGSrbj260oBAdKVu2RstgICJFtx7deVAgKkK3fJ2GwFBEi24tqvKwUESFfukrHZCgiQbMW1X1cKCJCu3CVjsxUQINmKa7+uFBAgXblLxmYrIECyFdd+XSkgQLpyl4zNVuBvJQ3KQHUW2IcAAAAASUVORK5CYII=';
```

Comment:
could you check if the following base64 files works fine? they should reduce the sdk /image size by 2kb